dm/master/election.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"github.com/pingcap/dm/pkg/log"
+)
+
+func (s *Server) electionNotify(ctx context.Context) {
```

Comment:
why need this function, seems `toBeLeader` and `retireLeader` already print the log

Reply:
`toBeLeader` and `retireLeader` only log the leadership of the current member.
this function is used to show the usage of `Election` (and also log the leader name even if it's not the current member), do you think should we need to remove it?

Reply:
one more thing, `pkg/election` is a pkg, but this function is in the application. Maybe we need to add some metrics for the leadership later, and do that in this function is better.

Reply:
if master needs to do something(for example: meet error in the election, master exit) according to the election's result I think putting here is better. 
it is up to you

Reply:
I chose to remove the log in `pkg/election` in https://github.com/pingcap/dm/pull/367/commits/b99ae4653c5039cfb5070e5b0d62d6bdad77599a.